test/e2e/subscription_e2e_test.go

```diff
@@ -2928,6 +2936,9 @@ var (
 	dummyCatalogConfigMap = &corev1.ConfigMap{
 		ObjectMeta: metav1.ObjectMeta{
 			Name: catalogConfigMapName,
+			Labels: map[string]string{
+				install.OLMManagedLabelKey: install.OLMManagedLabelValue,
+			},
```

Comment:
Is this label necessary now that we're using a live client?

Reply:
no. it's now